### PR TITLE
Sprint moosequery

### PR DESCRIPTION
**Pull request by mahugnon**
corrected comment.

### CODE REVIEW

**Comment by badetitou** [comment]
Is it the correction of the PR #1436 ? (why not only one PR...)
The title is incorrect, it is super annoying when we want to review.

File: src/Famix-Compatibility-Entities/FAMIXNamedEntity.class.st [hunk 1]
```diff
@@ -107,6 +107,18 @@ FAMIXNamedEntity >> stubFormattedName [
 		ifFalse: [ Text fromString: self name ] 
 ]
 
+{ #category : #'moosequery-queries-generic' }
+FAMIXNamedEntity >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-Compatibility-Entities/FAMIXNamedEntity.class.st]: The example and the comment are not correct

File: src/Famix-Compatibility-Entities/FAMIXSourceAnchor.class.st [hunk 2]
```diff
@@ -172,3 +172,15 @@ FAMIXSourceAnchor >> querySureOutgoingInvocations [
 FAMIXSourceAnchor >> sourceText [
 	^ self subclassResponsibility
 ]
+
+{ #category : #'moosequery-queries-generic' }
+FAMIXSourceAnchor >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-Compatibility-Entities/FAMIXSourceAnchor.class.st]: The example and the comment are not correct

File: src/Famix-Java-Entities/FamixJavaAnnotationInstance.class.st [hunk 3]
```diff
@@ -80,3 +80,15 @@ FamixJavaAnnotationInstance >> name [
 				<< (self annotationType ifNil: [ super name ] ifNotNil: [ :type | type name ])
 				<< ' on ' << (self annotatedEntity ifNotNil: #name ifNil: [ 'undefined' ]) ]
 ]
+
+{ #category : #'moosequery-queries-generic' }
+FamixJavaAnnotationInstance >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-Java-Entities/FamixJavaAnnotationInstance.class.st]: The example and the comment are not correct

File: src/Famix-Java-Entities/FamixJavaAnnotationInstanceAttribute.class.st [hunk 4]
```diff
@@ -64,3 +64,15 @@ FamixJavaAnnotationInstanceAttribute >> name [
 		ifTrue: [ self annotationTypeAttribute name ]
 		ifFalse: [ nil ]
 ]
+
+{ #category : #'moosequery-queries-generic' }
+FamixJavaAnnotationInstanceAttribute >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-Java-Entities/FamixJavaAnnotationInstanceAttribute.class.st]: The example and the comment are not correct

File: src/Famix-Java-Entities/FamixJavaContainerEntity.class.st [hunk 5]
```diff
@@ -77,3 +77,15 @@ FamixJavaContainerEntity >> numberOfChildren [
 	
 	^ self children size
 ]
+
+{ #category : #'as yet unclassified' }
+FamixJavaContainerEntity >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-Java-Entities/FamixJavaContainerEntity.class.st]: The example and the comment are not correct

File: src/Famix-Java-Entities/FamixJavaNamedEntity.class.st [hunk 6]
```diff
@@ -103,6 +103,18 @@ FamixJavaNamedEntity >> stubFormattedName [
 		ifFalse: [ Text fromString: self name ] 
 ]
 
+{ #category : #'moosequery-queries-generic' }
+FamixJavaNamedEntity >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-Java-Entities/FamixJavaNamedEntity.class.st]: The example and the comment are not correct

File: src/Famix-Java-Entities/FamixJavaSourceAnchor.class.st [hunk 7]
```diff
@@ -157,3 +157,15 @@ FamixJavaSourceAnchor >> querySureOutgoingInvocations [
 FamixJavaSourceAnchor >> sourceText [
 	^ self subclassResponsibility
 ]
+
+{ #category : #'moosequery-queries-generic' }
+FamixJavaSourceAnchor >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-Java-Entities/FamixJavaSourceAnchor.class.st]: The example and the comment are not correct

File: src/Famix-PharoSmalltalk-Entities/FamixStAnnotationInstance.class.st [hunk 8]
```diff
@@ -66,3 +66,15 @@ FamixStAnnotationInstance >> name [
 				<< (self annotationType ifNil: [ super name ] ifNotNil: [ :type | type name ])
 				<< ' on ' << (self annotatedEntity ifNotNil: #name ifNil: [ 'undefined' ]) ]
 ]
+
+{ #category : #'moosequery-queries-generic' }
+FamixStAnnotationInstance >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
```
Comment (badetitou) [inline src/Famix-PharoSmalltalk-Entities/FamixStAnnotationInstance.class.st]: The example and the comment are not correct

File: src/Famix-PharoSmalltalk-Entities/FamixStAnnotationInstanceAttribute.class.st [hunk 9]
```diff
@@ -64,3 +64,15 @@ FamixStAnnotationInstanceAttribute >> name [
 		ifTrue: [ self annotationTypeAttribute name ]
 		ifFalse: [ nil ]
 ]
+
+{ #category : #'moosequery-queries-generic' }
+FamixStAnnotationInstanceAttribute >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-PharoSmalltalk-Entities/FamixStAnnotationInstanceAttribute.class.st]: The example and the comment are not correct

File: src/Famix-PharoSmalltalk-Entities/FamixStContainerEntity.class.st [hunk 10]
```diff
@@ -43,3 +43,15 @@ FamixStContainerEntity >> numberOfChildren [
 	
 	^ self children size
 ]
+
+{ #category : #'as yet unclassified' }
+FamixStContainerEntity >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (badetitou) [inline src/Famix-PharoSmalltalk-Entities/FamixStContainerEntity.class.st]: The example and the comment are not correct

**Comment by jecisc** [comment]
You do not have to checkout in Pharo. You can just edit the comment from github since it's just comments.

**Pull request merged**